oc_cdtapi/VaultAPI.py

```
@@ -41,39 +40,59 @@ def client(self):
 
-    def parse_secret_name(self, name):
-        if 'USER' in name:
-            split_name = name.split('_USER')[0]
-            return split_name, 'USER'
-
-        if 'PASSWORD' in name:
-            split_name = name.split('_PASSWORD')[0]
-            return split_name, 'PASSWORD'
+    def _parse_secret_name(self, name: str) -> list[str]:
+        if not SECRET_PATTERN.match(name):
+            raise ValueError("Secret name must match <PATH>__<KEY>")
```

Ruff (0.14.8)

Avoid specifying long messages outside the exception class

(TRY003)